nltk/lm/api.py

```diff
@@ -162,6 +162,8 @@ def context_counts(self, context):
 
     def entropy(self, text_ngrams):
         """Calculate cross-entropy of model for given evaluation text.
+        This implementation is based on the Shannon-McMillan-Breiman theorem,
```

Comment:
```suggestion

        This implementation is based on the Shannon-McMillan-Breiman theorem,
```
Just to stay consistent with other docstrings in the module